flexmeasures/ui/templates/views/sensors.html

```diff
@@ -33,6 +33,29 @@
               <div class="alert alert-info" id="dstwarn" style="display:none;"></div>
               <div class="alert alert-info" id="sourcewarn" style="display:none;"></div>
           </div>
+          <div class="sensor-basic-info-container">
```

Comment:
I believe we need to pay attention to the bootstrap layout and fill in our new element in the right way.

For instance I would use `class="row"`, as the `div` above and below have done.

This is all part of [Boostrap's Layout](https://getbootstrap.com/docs/5.0/layout/grid/) which I advise to read - it's very helpful in achieving flexible box layouts without needing to know a lot of CSS. You might be able to remove the extra CSS you added above, even.

The next row has the Daterange selection aligned with the left edge of the screen, but this section should leave some whitespace on the left.

Reply:
By white space on the left side, do you mean like the below image?

![image](https://github.com/user-attachments/assets/c530a325-f284-4af8-935a-ccde9912640b)

Reply:
Yes. I mean the table is being rendered as much to the left edge of the screen as possible, which does not look good.

Reply:
After my recent change, the table is now centered. As you can see.

![image](https://github.com/user-attachments/assets/6faf8c85-2c72-4769-9afa-d5f277ea755c)